lib/pure/pathnorm.nim

```diff
@@ -100,15 +100,12 @@ proc addNormalizePath*(x: string; result: var string; state: var int;
   if result == "" and x != "": result = "."
 
 proc normalizePath*(path: string; dirSep = DirSep): string =
-  ## Example:
-  ##
-  ## .. code-block:: nim
-  ##   assert normalizePath("./foo//bar/../baz") == "foo/baz"
-  ##
-  ##
   ## - Turns multiple slashes into single slashes.
   ## - Resolves '/foo/../bar' to '/bar'.
   ## - Removes './' from the path (but "foo/.." becomes ".")
+  ##
+  runnableExamples:
+    doAssert normalizePath("./foo//bar/../baz") == "foo/baz"
```

Comment:
In windows `normalizePath("./foo//bar/../baz") == "foo\baz"`, the whole original comments are wrong.

Reply:
I will use `when defined(posix)`.